generate-code.sh

```diff
@@ -70,18 +75,22 @@ _heading "Preparing to generate library..."
 echo "- Schema Source Repository: \t${GENERATE_DATA_SOURCE} (${data_version} on ${data_date})"
 echo "- Proto Source Directory: \t${src_dir}"
 echo "- Shared googleapis Protos: \t${googleapis_dir}"
+echo "- Custom Code Generator Templates: \t${GENERATE_TEMPLATE_DIR}"
 
 # Manifest file with details about how code was most recently generated.
 # Useful when troubleshooting without build logs.
 cat > generated.txt <<GENERATION_METADATA
```

Comment:
Optional: consider adding this file to `.gitignore`, so it does not get accidentally committed.